.npmrc

```diff
@@ -1,4 +1 @@
-@fortawesome:registry=https://npm.fontawesome.com/
-//npm.fontawesome.com/C1F19C49-B88B-4523-8273-6C86A14E7F58=TOKEN
-//npm.fontawesome.com/:_authToken=C1F19C49-B88B-4523-8273-6C86A14E7F58
 //registry.npmjs.org/:_authToken=52d68335-e507-46a1-835f-c73f19279b12
```

Comment:
it is [this one](https://wekaio.atlassian.net/browse/WEKAPP-372274?atlOrigin=eyJpIjoiMjEwMzVmM2I5MzA0NGY3ZjkxMzU3NDljYTM3Y2RjMTIiLCJwIjoiaiJ9) we wanted to remove initially, no?